### ISSUE TITLE
ELI5 on how to use all those bindings

### ISSUE THREAD

**Issue by eyalmazuz** [kragen/xcompose#55]
I'm new to Xcompose and everything; I just found this repo with all those cool shortcuts; I know how to use <Multi_Key> to make some stuff, and I have a basic setup already in my Xcompose file e.g.
![image](https://user-images.githubusercontent.com/34383384/185355085-db99777c-29b1-4365-a477-15d2d88f63b9.png)

but I don't understand how to use maths-basic or any of the other files

what is the <M_> key means, and why {$#bK} in curly brackets and not <>
or what <MM> means etc. etc.

I couldn't find any explanation when searching google and I reached a point I found files like 
/usr/include/X11/keysymdef.h                                                                                                                                                                                      
/usr/share/X11/xkb/symbols/lv

but they were more confusing that helpful

is there a chance to have some links or explanation on how to use everything?

**Comment by clsn** [comment]
Yeah, those aren't standard files of any sort.  They're a format I made up to make it a little easier to write the compose files.  If you look, the Makefile runs the "base" files through a perl translator to make .compose files.  So I can write `<M_> {$#bK}` instead of `<Multi_key> <dollar> <numbersign> <b> <K>`, which is a lot more keystrokes.

**Comment by eyalmazuz** [comment]
I see, I didn't notice that
thank you very much
